### PR TITLE
All code files must now be included in the .dme, removes some old duplicate files that never were

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request
Adds a CI check to validate_dme.py to make sure every code file is included, with some explicit exceptions.

Doesn't supersede #67875 because it has some fixes I don't, merge that one first.

### CODE REVIEW

File: code/datums/ai/objects/item_behaviors.dm [hunk 1]
```diff
@@ -1,13 +0,0 @@
-
-///This behavior is for obj/items, it is used to free themselves out of the hands of whoever is holding them
-/datum/ai_behavior/item_escape_grasp
-
-/datum/ai_behavior/item_escape_grasp/perform(delta_time, datum/ai_controller/controller)
```
Comment (Iamgoofball) [inline code/datums/ai/objects/item_behaviors.dm:5]: These should be migrated to an unused dm and checked, these are useful.

Comment (Mothblocks) [reply code/datums/ai/objects/item_behaviors.dm:5]: Useful for what, this is just a copy of something that exists in another file

Comment (Iamgoofball) [reply code/datums/ai/objects/item_behaviors.dm:5]: wait why wasn't this erroring on duplicate defines already

Comment (MrDoomBringer) [reply code/datums/ai/objects/item_behaviors.dm:5]: because this file was unincluded in the tgstation.dme

Comment (Mothblocks) [reply code/datums/ai/objects/item_behaviors.dm:5]: What MDB said 😄

**Pull request merged**